websites/R/RTL+/presence.ts

```diff
@@ -0,0 +1,154 @@
+					episode = footerEles[4]
+						?.querySelector("span")
+						?.textContent.split(" Folge ");
+
+				title = footerEles[2]?.querySelector("a>span")?.textContent.trim();
+
+				presenceData.state = `S${season}:E${episode?.[1]?.trim()} ${episode?.[0]?.trim()}`;
+				presenceData.buttons[0].label = "Watch Series";
+			} else if (pathname.includes("/filme/")) {
+				title = footerEles[2]?.querySelector("a>span")?.textContent.trim();
+				presenceData.buttons[0].label = "Watch Movie";
+			}
+
+			if (!title) {
+				title = document.title
+					.replace("im Online Stream | RTL+", "")
+					.replace("im Online Stream ansehen | RTL+", "")
+					.trim();
+			}
+
+			if (cover) presenceData.largeImageKey = cover.src;
+
+			presenceData.details = title;
+
+			if (duration <= 5) {
+				delete presenceData.startTimestamp;
+				delete presenceData.endTimestamp;
+
+				presenceData.smallImageKey = "https://i.imgur.com/C6mbMYz.png";
+				presenceData.smallImageText = (await strings).pause;
+			}
+		}
+	} else if (pathname === "/") {
+		presenceData.details = "Viewing main page";
+		presenceData.startTimestamp = browsingTimestamp;
+	} else if (pathname.includes("/serien/")) {
+		presenceData.details = "Viewing series:";
+		presenceData.state =
+			document.querySelector(".ng-tns-c96-53")?.textContent ??
+			document.title.replace("im Online Stream ansehen | RTL+", "").trim();
+		presenceData.startTimestamp = browsingTimestamp;
+
+		presenceData.buttons = [
+			{
+				label: "View Series",
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/shows/")) {
+		presenceData.details = "Viewing show:";
+		presenceData.state =
+			document.querySelectorAll("li.ng-star-inserted span.ng-star-inserted")[1]
+				?.textContent ??
+			document.title.replace("im Online Stream ansehen | RTL+", "").trim();
+		presenceData.startTimestamp = browsingTimestamp;
+
+		presenceData.buttons = [
+			{
+				label: "View Show",
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/serien")) {
+		presenceData.details = "Browsing series";
+		presenceData.startTimestamp = browsingTimestamp;
+	} else if (pathname.includes("/shows")) {
+		presenceData.details = "Browsing shows";
+		presenceData.startTimestamp = browsingTimestamp;
+	} else if (pathname.includes("/filme/")) {
+		presenceData.details = "Viewing movie:";
+		presenceData.state =
+			document.querySelectorAll("li.ng-star-inserted span.ng-star-inserted")[1]
+				?.textContent ??
+			document.title.replace("im Online Stream ansehen | RTL+", "").trim();
+		presenceData.startTimestamp = browsingTimestamp;
+		presenceData.buttons = [
+			{
+				label: "View Movie",
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/filme")) {
+		presenceData.details = "Browsing movies";
+		presenceData.startTimestamp = browsingTimestamp;
+	} else if (pathname.includes("/specials/")) {
+		presenceData.details = "Viewing special:";
+		presenceData.state = document.title;
+		presenceData.startTimestamp = browsingTimestamp;
```

Comment:
Also remove all of these put it at the presenceData and just delete it when a video is playing